src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,47 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Book {
+
+    @Id
+    private long id;
+
+    private String genre;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false, name = "sub_title")
+    private String subTitle;
```

Comment:
I prefer to think subtitle as a one word, not two. But it's up to you to choose the option that you like the most :) .

Reply:
Solved.